core/src/services/etcd/backend.rs

```diff
@@ -303,22 +302,6 @@ impl kv::Adapter for Adapter {
         let _ = client.delete(p, None).await?;
         Ok(())
     }
-
-    async fn scan(&self, path: &str) -> Result<Vec<String>> {
```

Comment:
why do you delete `scan` function?

Reply:
Because the integration test could not pass. I tried to fix it but failed, can you give me some suggestions?